flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CommonUtils.java

```diff
@@ -507,12 +501,10 @@ public static String buildResourcePathTemplate(FunctionSymbol functionSymbol) {
                     }
                 }
                 ((PathSegmentList) resourcePath).pathRestParameter().ifPresent(pathRestParameter -> {
-                    String type = CommonUtil.getRawType(pathRestParameter.typeDescriptor())
-                            .signature();
-                    pathBuilder.append("[").append(type).append("...]");
+                    pathBuilder.append("/path/to/subdirectory");
                 });
             }
-            case PATH_REST_PARAM -> pathBuilder.append("[").append("/path/to/resource").append("]");
+            case PATH_REST_PARAM -> pathBuilder.append("/path/to/subdirectory");
```

Comment:
why subdirectory ?

Reply:
resource is an escaped keyword. In url conventions they call it sub directory

Reply:
OK. That makes sense.